templates/shortcode-profile-editor.php

```diff
@@ -26,7 +26,10 @@
 
 	<?php do_action( 'edd_profile_editor_before' ); ?>
 
-	<form id="edd_profile_editor_form" class="edd_form" action="<?php echo edd_get_current_page_url(); ?>" method="post">
+	<form id="edd_profile_editor_form" class="edd_form" action="<?php echo edd_get_current_page_url(); ?>" method="post" <?php do_action( 'edd_profile_editor_form_tag' ); ?>>
```

Comment:
@SDavisMedia @spencerfinnell @rubengc I find it odd to have an action within an HTML tag. The only way this would be useful is to echo out information into the tag, but that's kind of not an ideal situation as there is no guarentee that the person using the hook is going to properly escape, or do proper sanitization of the form field tag.

I'm not sure I see the point of allowing someone to openly run an action on the attributes of a form field, unless I'm missing something.